src/convertToObject.js

```diff
@@ -3,16 +3,25 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  sourceString
+    .split(';')
+    .filter(string => string.trim())
+    .map(string => string.split(':'))
+    .forEach(string => (styleObject[string[0].trim()] = string[1].trim()));
```

Comment:
Create meaningful variables, `string` here - is not a good option. Variable name should describe what inside it.  `styleString` for e[ample would be better.

Reply:
Variable names changed.
Thanks for review)